frontend/src/muralForm/MuralForm.tsx

```diff
@@ -271,6 +272,41 @@ function MuralForm({ mural, handleCancel }: IMuralFormProps) {
     return null;
   }
 
+  function copy() {
```

Comment:
maybe name this function name a bit more specific so there's better readability? nitpick tho

Reply:
changed